### PR TITLE
Accounts Benefits Page (#6339)

### PR DESCRIPTION
**Pull request by amychurchwell**
## Description
New /firefox/accounts/ page.

## Issue / Bugzilla link
#6339

## Testing
https://bedrock-demo-achurchwell.oregon-b.moz.works/en-US/firefox/accounts/

### CODE REVIEW

**Comment by amychurchwell** [comment]
@dzingeek Jennifer's creative feedback before she left on PTO:

- [ ] Links underlined and blue to further illustrate they are clickable links.
- [ ] Blue firefox qr button instead of black.
- [ ] Change Zilla Slab to sans-serif Firefox brand font.

**Comment by craigcook** [comment]
@peiying2 These strings are complete and frozen, ready to expose for l10n, though there's still some more front-end work to finish up the page.

**Comment by peiying2** [comment]
@craigcook Thanks for the update.

File: bedrock/privacy/templates/privacy/faq.html [hunk 1]
```diff
@@ -206,8 +206,8 @@ <h2>
             {%- endtrans -%}
           </p>
           <p>
-            {%- trans -%}
-              You don't need an account to use Firefox.  Accounts are required to sync data across devices, but we only ask you for an email address.  We don't want to know things like your name, address, birthday and phone number.
+            {%- trans accounts=url('firefox.accounts') -%}
```
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/privacy/templates/privacy/faq.html]: Oh crap, sorry, I kept thinking this page was replacing /accounts/features/ which was never localized so we weren't worried about breaking strings. But now I see it's replacing /accounts/ which *is* localized so we need to rethink this... sorry for the confusion.

I think we should spin off a new template with a new lang file and let locales fall back to the old page until they activate the new one. I'll set that up and close this PR and open a fresh one in its place.

**Comment by craigcook** [comment]
Closing this PR in favor of #6503 

@amychurchwell Because the page we're replacing is localized we want to allow some time for slower locales to update to the new page. So I've spun it off into its own template called `accounts-2018.html` and other locales can fall back to the old page until they have time to translate the new one. Eventually we'll be able to truly delete the old page and any locales that are still behind will just get English, but this gives them more time to work (though it does also incur some new tech debt; such is life).

I've put the new branch in the mozilla repo rather than in my fork so you can commit to it while you work on finishing it up. Hopefully I haven't broken anything too badly.

File: bedrock/privacy/templates/privacy/faq.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (craigcook) [reply bedrock/privacy/templates/privacy/faq.html]: Also just realized my comment above was really in response to the review summary comment, not this particular line... it's been a weird day.

**Pull request closed without merging**